client-js/app/elements/labrad-grapher-dataset.html

```diff
@@ -18,15 +30,17 @@
   Polymer({
     is: 'labrad-grapher-dataset',
     properties: {
-      items: {
-        type: Array,
+      path: {
+        type: String,
         notify: true,
+      },
+      dataset: {
+        type: String,
+        notify: true
       }
     },
-    ready: function() {
-      this.items = [
-        'A single dataset'
-      ];
+    back: function(){
+      window.location.href='/grapher/'+this.path.toString().replace(/,/g,"/");
```

Comment:
well, this is new logic (being able to navigate back to the directory listing), so I'd say let's just fix it in `loadDataset`